game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/README.md

```diff
@@ -0,0 +1,94 @@
+# ETS2 Ptero - A Egg For Setting Up ETS2 Dedicated Server
+
+
+Ports below are the default ports that are associated with the ETS2 dedicated server.  If you change them in server setup be sure to change them in the .local/share/Euro Truck Simulator 2/server_config.sii file. (See Server_Config Directions below)
+| Port     | default       |
+|----------|---------------|
+| Dedicated| 27015         |
+| Query    | 27016         |
+
+--**Make sure during server setup that the autostart server box is unchecked**--
+
+# Setup
+You must generate the required server config files necessary to run the server in game on ETS2.  To do so you **MUST** have console enabled.  
+at
+# Enabling Console
+To enable your console you must change 2 values in the clients config.cfg file.  With the game off simply change the following:
+- g_developer "0" -----> g_developer "1"
+- g_console "0" -----> g_console "1"
+
+Once those have been changed save and close the file and start up ETS2.
+
+# Generating Config Files
+Once you have enabled your console and started ETS2, go ahead and log in to game and set up the convoy how you would normally set it up.  This includes, any mods that you may have.  Also, dont worry about setting up the name, password, description, etc, as the server_config.sii generated will ignore these settings.  
+
+Once you have your convoy set up in game, press the ~ key and a console will open on the bottom center of your screen.  If a console does not open, close ETS2 and go back up to **Enabling Console** section and go through those steps again. 
+
+With the console open type ```export_server_packages``` and press enter.  The game will then tell you that the export was successful.  
+
+Navigate to the folder that contains your clients config.cfg file that you changed in the **Enabling Console** section and look for the following:
+
+- server_packages.sii
+- server_packages.dat
+
+
+# Copy to Server
+
+With your newly set up files its time to copy them over to the Pterodactyl Panel.  Simply upload them through either your preferred SCP program or the file browser in the server panel to the .local/share/Euro Truck Simulator 2/ folder.
+
+On the panel select start server and the server will then start up and create a default server_config.sii file.  Proceed to the .local/share/Euro Truck Simulator 2/folder.

```

Comment:
there is no such a dir: `.local/share/Euro Truck Simulator 2/`

there is only :
![chrome_Tp8F2uFhP3](https://user-images.githubusercontent.com/116325/206904527-dc841cf2-7521-4784-80be-94cd9037a477.png)
![chrome_XvuvsTvegl](https://user-images.githubusercontent.com/116325/206904530-9a0427bd-2dc5-4531-a28a-c54333171ae8.png)

Reply:
it should be generate on install script